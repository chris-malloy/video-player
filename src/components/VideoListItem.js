import React from 'react';

export default function VideoListItem() {
    return (
        <div>hi</div>
    )
}