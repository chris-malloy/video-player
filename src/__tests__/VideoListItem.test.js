import React from 'react';
import VideoListItem from '../components/VideoListItem';
import renderer from 'react-test-renderer';

describe('Component: VideoListItem', () => {

    test('VideoListItem renders correctly', () => {
        const component = renderer.create(
            <VideoListItem>VideoListItem</VideoListItem>
        )
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
});