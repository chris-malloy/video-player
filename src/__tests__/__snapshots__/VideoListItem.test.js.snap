// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: VideoListItem VideoListItem renders correctly 1`] = `
<div>
  hi
</div>
`;
